manim/scene/scene_file_writer.py

```diff
@@ -359,6 +362,33 @@ def end_animation(self, allow_write: bool = False):
         if write_to_movie() and allow_write:
             self.close_partial_movie_stream()
 
+    def listen_and_write(self):
+        """
+        For internal use only: blocks until new frame is available on the queue.
+        """
+        while True:
+            num_frames, frame_data = self.queue.get()
+            if frame_data is None:
+                break
+
+            self.encode_and_write_frame(frame_data, num_frames)
+
+    def encode_and_write_frame(self, frame: PixelArray, num_frames: int) -> None:
+        """
+        For internal use only: takes a given frame in `np.ndarray` format and
```

Comment:
```suggestion
        For internal use only: takes a given frame in ``np.ndarray`` format and
```
Oops, sorry I missed this last time. Sphinx parses this as italics, so you have to put two backticks to make it inline code.